clash-prelude/src/Clash/Sized/Internal/BitVector.hs

```diff
@@ -1237,14 +1237,14 @@ undefined# =
 --
 -- >>> let expected = $$(bLit "1.") :: BitVector 2
 -- >>> let checked  = $$(bLit "11") :: BitVector 2
--- >>> checked  `isLike` expected
+-- >>> checked  `isLike#` expected
 -- True
--- >>> expected `isLike` checked
+-- >>> expected `isLike#` checked
```

Comment:
As you linked to it in `isLike` I read the documentation of this function. I think it could use some work:

* I think we should introduce it as "Checks for equality but ignores undefined bits in its second argument". I think "Check if one BitVector is like another." is too vague/imprecise. 
* Explicitly mention what happens when bits are set to undefined in the _first_ argument.

Reply:
> "Check if one BitVector is like another." is too vague/imprecise

At least it's correct :wink: 

But no, point taken on the documentation here and in the other module. I'll make it better tomorrow